pkg/kv/kvserver/raft_transport.go

```diff
@@ -864,6 +903,16 @@ func (t *RaftTransport) SendAsync(
 		}
 	}
 
+	outgoingMessageHandler, ok := t.getOutgoingMessageHandler(req.FromReplica.StoreID)
+	if ok {
+		outgoingMessageHandler.HandleRaftRequestSent(t.AnnotateCtx(context.Background()), req)
```

Comment:
`AnnotateCtx` is very expensive, just pass `context.Background` or even better remove the context argument.

Reply:
Done. Used `context.Background` - can't easily remove the context argument since `getLocalityComparison` needs error logging.